cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -358,3 +394,93 @@ func (r *KubeArchiveConfigReconciler) desiredA14e(kaconfig *kubearchivev1alpha1.
 
 	return source, nil
 }
+
+func (r *KubeArchiveConfigReconciler) reconcileFilterConfigMap(ctx context.Context, kaconfig *kubearchivev1alpha1.KubeArchiveConfig) (*corev1.ConfigMap, error) {
+	log := log.FromContext(ctx)
+
+	log.Info("in reconcileFilterConfigMap")
+
+	cm := &corev1.ConfigMap{}
+	err := r.Get(ctx, types.NamespacedName{Name: SinkFilterConfigMapName, Namespace: k9eNs}, cm)
+	if err == nil {
+		cm, err := r.desiredFilterConfigMap(ctx, kaconfig, cm)
+		if err != nil {
+			log.Error(err, "Unable to get desired ConfigMap "+SinkFilterConfigMapName)
+			return cm, err
+		}
+		err = r.Update(ctx, cm)
+		if err != nil {
+			log.Error(err, "Failed to update filter ConfigMap "+SinkFilterConfigMapName)
+			return cm, err
+		}
+	} else if errors.IsNotFound(err) {
+		cm, err := r.desiredFilterConfigMap(ctx, kaconfig, nil)
+		if err != nil {
+			log.Error(err, "Unable to get desired filter ConfigMap "+SinkFilterConfigMapName)
+			return cm, err
+		}
+		err = r.Create(ctx, cm)
+		if err != nil {
+			log.Error(err, "Failed to create filter ConfigMap "+SinkFilterConfigMapName)
+			return cm, err
+		}
+	} else {
+		log.Error(err, "Failed to reconcile filter ConfigMap "+SinkFilterConfigMapName)
+		return cm, err
+	}
+
+	return cm, nil
+}
+
+func (r *KubeArchiveConfigReconciler) desiredFilterConfigMap(ctx context.Context, kaconfig *kubearchivev1alpha1.KubeArchiveConfig, cm *corev1.ConfigMap) (*corev1.ConfigMap, error) {
+	log := log.FromContext(ctx)
+
+	if cm == nil {
+		cm = &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      SinkFilterConfigMapName,
+				Namespace: k9eNs,
+			},
+			Data: map[string]string{},
+		}
+	}
+
+	jsonBytes, err := json.Marshal(kaconfig.Spec.Resources)
```

Comment:
PTAL to my last comment in the issue: https://github.com/kubearchive/kubearchive/issues/221#issuecomment-2252643480

Reply:
Yeah, I replied there. The YAML suggest is great.  I'll try that.

Reply:
Fixed.